LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 CORE
```

Comment:
```suggestion
Copyright (c) 2020 CORE Team <theteam@core.ac.uk>
```

I am not sure about this, we should consult with whole team @oacore/services. I used 'Knowledge Media Institute, The Open University' before but I am not sure about this neither.